### DISCUSSION TITLE
What is scientific name of diamond💎  ?

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#23] [Galaxy Qs] [answered]
What is scientific name of diamond💎  ?

**Comment by bytesleo** [comment, marked as answer]
The scientific name of diamond is "carbon" or "carbonado." Diamond is a form of carbon that is known for its crystal structure and extreme hardness. Its chemical composition is pure carbon, and it is a naturally occurring mineral. Carbonado is a type of diamond that is polycrystalline in nature and is commonly found in Brazil and Central Africa.
